README.md

```diff
@@ -425,21 +323,36 @@ You can lint your code independently of this by running:
+
+We structure our logs according to these [NYPL standards](https://github.com/NYPL/engineering-general/blob/main/standards/logging.md). See `logger.js` for the formatting and storage of logs.
+
+Do NOT use `console.log` if you want to add a permanent log, and make sure to clean up your debugging `console.log`s, because they will clutter Cloudwatch and New Relic.
+
+On the server side, DO use `logger.[some NYPL log level](message: string)`.  
+On the client side, DO use `newrelic.log(message: string, {level: 'debug|error|info|trace|warn'})` OR `newrelic.error(message: string)` (New Relic log levels only partially correspond to NYPL log levels).
+
 ## Github Actions
 
 All pushes to this repo will be checked with `npm test` and `npm lint`.
+QA deployments are automatically triggered by pushing changes to the qa branch. qa is deployed to AWS via Github Actions.
+Production deployments for this repo require a PR to the production branch. Once merged, production is deployed to AWS via Github Actions.
+
+## Branches and AWS environments
 
-## Git Workflow
+This app is currently on a reverse proxy with ["old" Digital Collections](https://github.com/NYPL/digitalreadingroom), which is no longer being updated. The [reverse proxy repo](https://github.com/NYPL/nypl-dc-rp) configuration sets which paths are redirected to old and new (for QA and production).
 
 Our branches (in order of stability are):
 
-| Branch     | Environment | AWS Account    | Cluster                | Link To Application                                                                        |
-| :--------- | :---------- | :------------- | :--------------------- | :----------------------------------------------------------------------------------------- |
-| main       | development |                |                        | [localhost:3000](http://localhost:3000)                                                    |
-| qa         | qa          | nypl-dams-dev  | dc-frontend-qa         | [https://qa-new-digitalcollections.nypl.org/](https://qa-new-digitalcollections.nypl.org/) |
-| production | production  | nypl-dams-prod | new-digitalcollections | [https://new-digitalcollections.nypl.org/](https://new-digitalcollections.nypl.org/)       |
+| Branch     | Environment | AWS Account    | Cluster                                       | Link To Application                                                                    |
+| :--------- | :---------- | :------------- | :-------------------------------------------- | :------------------------------------------------------------------------------------- |
+| main       | development |                |                                               | [localhost:3000](http://localhost:3000)                                                |
+| qa         | qa          | nypl-dams-dev  | nyplorg-dc-rp-qa AND dc-frontend-qa           | [https://qa-digitalcollections.nypl.org/](https://qa-new-digitalcollections.nypl.org/) |
```

Comment:
Is this confusing to list two clusters/ is this technically correct? I think the fact that I'm STILL unsure about this setup means we need to have a very clear written description so I'd like to make that happen

Reply:
That's fair and I can include that later, but no, this is not technically correct. The cluster is only `dc-frontend-qa` (similar for prod). The reverse proxy cluster is its own thing.